.github/workflows/release.yml

```
@@ -15,14 +15,20 @@ jobs:
     runs-on: ubuntu-latest
     name: "Release a new version"
     steps:
+      - name: Get auth token
+        id: token
+        uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 #v.1.11.0
```

```suggestion
        uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 #v1.11.0
```

ok maybe it doesn't look weird to you but this is what I mean